modules/core/04-channel/keeper/packet_test.go

```diff
@@ -855,31 +851,23 @@ func (suite *KeeperTestSuite) TestAcknowledgePacket() {
 			suite.Require().NoError(err)
 			channelCap = suite.chainA.GetChannelCapability(path.EndpointA.ChannelConfig.PortID, path.EndpointA.ChannelID)
 		}, false},
-		// {"channel in ackupgrade and flush status flush complete", func() {
-		// 	expError = types.ErrInvalidChannelState
-
-		// 	suite.coordinator.Setup(path)
-		// 	packet = types.NewPacket(ibctesting.MockPacketData, 1, path.EndpointA.ChannelConfig.PortID, path.EndpointA.ChannelID, path.EndpointB.ChannelConfig.PortID, path.EndpointB.ChannelID, defaultTimeoutHeight, disabledTimeoutTimestamp)
-		// 	channelCap = suite.chainA.GetChannelCapability(path.EndpointA.ChannelConfig.PortID, path.EndpointA.ChannelID)
-
-		// 	// Send a packet on B to disallow channel automatically moving to OPEN on UpgradeAck
-		// 	sequence, err := path.EndpointB.SendPacket(defaultTimeoutHeight, disabledTimeoutTimestamp, ibctesting.MockPacketData)
-		// 	suite.Require().Equal(uint64(1), sequence)
-		// 	suite.Require().NoError(err)
+		{"channel in flush complete state", func() {
```

Comment:
added back this commented out testcase and refactored to now set the state directly as flush complete.

I think its easier to rely on setting the state directly in these tests rather than calling the entire upgrade handshake flow with `path.Endpoint.ChanUpgradeInit` etc